백세희/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>내가 소개하는 오혜성님</title>
+    <link href="index.css"  type="text/css" rel="stylesheet"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Single+Day&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+    <header>
+        <div>
+            <img src="./KakaoTalk_20221102_050937282_02.jpg" width="300px" height="300px" alt="오혜성님의 사진"><br>
+            <h1><span>Oh!</span>혜성<br>
+                haesungoh414@gmail.com</h1>   
+        </div>
+    </header>
+    
+    <main>
+        <section>
+            <article class="introduce">
+                <div>
+                    <img src="./KakaoTalk_20221102_054244479_02.jpg" width="350px" height="270px" alt="전공 소개" >
+                    <p>전공은 <span class="emphasize">소프트웨어학과</span>,
+                    부전공은 <span class="emphasize">정보통신학과</span>입니다.
+                    </p>
+                </div>
+            </article>
+      
+            <article class="introduce">
+                <div>
+                    <img src="./KakaoTalk_20221102_050937282_01.jpg" width="350px" height="270px"  alt="강아지">
+                    <p><span class="emphasize">강아지</span>를 아주 사랑합니다. 
```

Comment:
```suggestion
                    <p><strong>강아지</strong>를 아주 사랑합니다. 
```
```css
strong {
  color: pink
}
```

이런 류의 태그도 있으니 확인해 보시면 좋을듯요~
https://developer.mozilla.org/ko/docs/Web/HTML/Element/strong

Reply:
네 확인해 보겠습니다!